job_templates/packaging_job.xml.em

```diff
@@ -134,7 +134,7 @@ if [ -n "${CI_AMENT_TEST_ARGS+x}" ]; then
   if [[ "$CI_AMENT_TEST_ARGS" != *-- ]]; then
     CI_AMENT_TEST_ARGS="$CI_AMENT_TEST_ARGS --"
   fi
-  export CI_ARGS="$CI_ARGS --ament-test-args $CI_AMENT_TEST_ARGS"
+  export CI_ARGS="$CI_ARGS --ament-test-args $CI_AMENT_TEST_ARGS --"
```

Comment:
Good point this should probably match the same code in the non-packaging jobs: https://github.com/ros2/ci/blob/eace1b0f2600222a26c75fbf919593d6cf06ee3f/job_templates/ci_job.xml.em#L149